src/Tabs/index.js

```diff
@@ -36,7 +42,9 @@ const Tabs = ({ children, defaultSelectedIndex = 0, onTabChange = noop }) => {
         changeTabs,
       }}
     >
-      <div className="nds-tabs">{children}</div>
+      <div className={cc(["nds-tabs", { "nds-tabs--bordered": hasBorder }])}>
```

Comment:
it really makes me happy that `classcat` exists (also i know `cat` is probably short for `concatenation` but still 🐈 )